src/datajudge/constraints/stats.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any, Optional, Tuple
+
+import sqlalchemy as sa
+from scipy.stats import ks_2samp
+
+from datajudge import Constraint
```

Comment:
```suggestion
from .base import Constraint, OptionalSelections
```
Adapting for consistency. The other modules in `/constraints/` also import relative to the current module.